app/Filament/Admin/Resources/Users/UserResource.php

```diff
@@ -119,44 +144,318 @@ public static function defaultTable(Table $table): Table
+                                    ])
+                                    ->required()
+                                    ->prefixIcon('tabler-clock-pin')
+                                    ->default('UTC')
+                                    ->selectablePlaceholder(false)
+                                    ->options(fn () => collect(DateTimeZone::listIdentifiers())->mapWithKeys(fn ($tz) => [$tz => $tz]))
+                                    ->searchable()
+                                    ->native(false),
+                                Select::make('language')
+                                    ->label(trans('profile.language'))
+                                    ->columnSpan([
+                                        'default' => 1,
+                                        'md' => 1,
+                                        'lg' => 1,
+                                    ])
+                                    ->required()
+                                    ->prefixIcon('tabler-flag')
+                                    ->live()
+                                    ->default('en')
+                                    ->searchable()
+                                    ->selectablePlaceholder(false)
+                                    ->options(fn (LanguageService $languageService) => $languageService->getAvailableLanguages())
+                                    ->native(false),
+                                FileUpload::make('avatar')
+                                    ->visible(fn (FileUpload $fileUpload, User $user) => $fileUpload->getDisk()->exists($fileUpload->getDirectory() . '/' . $user->id . '.png'))
+                                    ->avatar()
+                                    ->directory('avatars')
+                                    ->disk('public')
+                                    ->formatStateUsing(function (FileUpload $fileUpload, User $user) {
+                                        $path = $fileUpload->getDirectory() . '/' . $user->id . '.png';
+                                        if ($fileUpload->getDisk()->exists($path)) {
+                                            return $path;
+                                        }
+                                    })
+                                    ->deleteUploadedFileUsing(function (FileUpload $fileUpload, $file) {
+                                        if ($file instanceof TemporaryUploadedFile) {
+                                            return $file->delete();
+                                        }
+
+                                        if ($fileUpload->getDisk()->exists($file)) {
+                                            return $fileUpload->getDisk()->delete($file);
+                                        }
+                                    }),
+                                Section::make(trans('profile.tabs.oauth'))
+                                    ->visible(fn (User $user) => filled($user->oauth))
```

Comment:
I feel like this should be always be visible for the sake of UI/ UX consistency.